app/retail/templates/shared/head.html

```diff
@@ -76,9 +76,13 @@
 <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
 {% if meta_title %}<meta name="title" content="{{meta_title}}" />{% endif %}
 {% if meta_description %}<meta name="description" content="{{meta_description}}" />{% endif %}
+<script>
+  window.fortmatic_api_key = '{{ fortmatic_api_key }}'
+</script>
+<script async src="{% static "v2/js/lib/watch.js" %}"></script>
+<script async src="{% static "v2/js/lib/web3-0.20.7.min.js" %}"></script>
```

Comment:
we are using 0.20.6 on shared_scripts 20.7 is with some issues.

Reply:
also we already load it on shared_scripts.html

Reply:
@fvictorio ^

Reply:
Hi, I cannot push to this branch, but assuming `web3` is present in all the relevant pages, this line and the file it includes could just be removed (I did it locally but couldn't test it because i'm getting a `No module named 'celery'` error when I start docker compose).

Reply:
@fvictorio I'll push the changes over to your branch and you can take it forward from there

Reply:
> i'm getting a No module named 'celery' error when I start docker compose).

you need to pip install the latest requirements file

Reply:
Shouldn't this be done when the image is built?

Reply:
Oh, my bad. I forgot that you need to do `docker-compose build` explicitly.

Reply:
it is; but i think the upstream branch got updated + this was pulled in after you built